nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -883,6 +929,11 @@ in
           Otherwise, we recommend
 
             ${opt.writableStore} = false;
+            ''
+      ++ optional (cfg.directBoot.enable && cfg.useBootLoader)
```

Comment:
Should this maybe just be forbidden, i.e. asserted? I can't think of a situation where this is sensible.

Reply:
Technically you could want to direct boot then reboot using the bootloader. But we don't have a test driver that can dynamically offer the right API for that yet.